examples/loader-datawrapper/src/index.md

```diff
@@ -0,0 +1,83 @@
+# Datawrapper data loader
+
+Here’s a Python data loader that generates an HTML [iframe](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe) tag that embeds a [Datawrapper](https://www.datawrapper.de/) chart.
+
+```python
+import sys
+
+import pandas as pd
+from datawrapper import Datawrapper
+
+# Get an extract of arrests made by the Baltimore Police Department
+df = pd.read_csv(
+    "https://raw.githubusercontent.com/palewire/first-automated-chart/main/_notebooks/arrests.csv",
+    parse_dates=["ArrestDateTime"]
+)
+
+# Create a year column
+df['year'] = df.ArrestDateTime.dt.year
+
+# Calculate the total number by year
+totals_by_year = df.year.value_counts().sort_index().reset_index()
+
+# Connect to the datawrapper api
+dw = Datawrapper()
+
+# Create a chart
+chart_config = dw.create_chart(
+    # Headline the chart
+    title="Baltimore Arrests",
+    # Set the type
+    chart_type="column-chart",
+    # Give the data
+    data=totals_by_year,
+    # Configure other bits
+    metadata={
+        # Set the description
+        "describe": {
+            "source-name": "OpenBaltimore",
+            "source-url": "https://data.baltimorecity.gov/datasets/baltimore::bpd-arrests/about",
+        }
+    }
+)
+
+# Pull out the chart's unique identifier
+chart_id = chart_config["id"]
+
+# Publish the chart
+dw.publish_chart(chart_id)
+
+# Write the chart's embed code to stdout
+html = dw.get_iframe_code(chart_id, responsive=True)
+sys.stdout.write(html)
+```
+
+<div class="note">
+
+To run this data loader, you will need to create an API token in Datawrapper and set it as an environment variable named `DATAWRAPPER_ACCESS_TOKEN`.
+
+You’ll also need python3 and the pandas and datawrapper modules installed and available on your `$PATH`.
```

Comment:
this is what the command above ensures. Is there a way to make this easier by conflating this note and the tip below into a single procedure?